code/modules/power/supermatter/supermatter.dm

```diff
@@ -381,21 +382,20 @@
 
 	if(damage > warning_point) // while the core is still damaged and it's still worth noting its status
 		if((REALTIMEOFDAY - lastwarning) / 10 >= WARNING_DELAY)
-			var/stability = num2text(round((damage / explosion_point) * 100))
 
 			if(damage > emergency_point)
-				radio.talk_into(src, "[emergency_alert] Instability: [stability]%", common_channel, get_spans(), get_default_language())
+				radio.talk_into(src, "[emergency_alert] Integrity: [integrity]%", common_channel, get_spans(), get_default_language())
```

Comment:
these should be ```get_integrity()``` and ```integrity``` should be a local var again, not on the object like you've made it.
Otherwise there's a risk of out of data info.